packages/warriorjs-core/src/Unit.js

```diff
@@ -207,6 +211,18 @@ class Unit {
     this.score += points;
   }
 
+  /**
+   * Substract the given points from the score.
```

Comment:
Small typo here: "Substract" should be "subtract".

Reply:
Wow, I've been writing it wrong all these years...